TradeItIosEmsApi/TradeItTransactionsHistoryResult.h

```diff
@@ -0,0 +1,13 @@
+#import "TradeItResult.h"
+
+@class TradeItTransactionHistory;
+
+@protocol TradeItTransactionHistory
+
+@end
+
+@interface TradeItTransactionsHistoryResult : TradeItResult
+
+@property(nonatomic, copy, nullable) NSArray<TradeItTransactionHistory*> <Optional, TradeItTransactionHistory>  * transactionHistoryDetailsList;
```

Comment:
Is the array actually nullable? Why?

Reply:
No, but I put all optional to avoid blowing up in case the server returns null

Reply:
The server should not return `null` here ever. Do we have an E2E test that asserts this returns an array?

Reply:
@mitochondrion thoughts?